cmd/check.go

```diff
@@ -45,7 +44,7 @@ Flags:
 			return types.Error{Msg: "Unable to create OpCap client."}
 		}
 		var packageManifestList pkgserverv1.PackageManifestList
-		err = psc.ListPackageManifests(context.TODO(), &packageManifestList, checkflags)
+		err = psc.ListPackageManifests(context.TODO(), &packageManifestList, checkflags.CatalogSource, checkflags.FilterPackages)
```

Comment:
This one is a little strange to me. This is requiring a remote call in a PreRun, that doesn't seem correct. This probably goes away with #185.

Reply:
Agreed. Left a comment below to change the function signature as well. @bcrochet

Reply:
Right @bcrochet , I think that one can be done under the other issue. Thanks for catching that one!